docs/SSO/Identity-Provider-SSO.md

```diff
@@ -2,23 +2,49 @@
 
 You can replace Vendasta's login screens with your own by configuring Identity Provider (IDP) SSO. It is a great option if your customers already have a username & password (or other credentials) stored in your system. 
 
-We currently support either of the OAuth2 or OpenIDConnect standardized authentication flows.
+Vendasta currently supports OpenIDConnect, an industry standard protocol built to extend OAuth2.0 with a user identification layer that enables secure authentication and user identity verification in addition to the authorization flow provided by OAuth2.0.
 
-If you are using a commercial system to manage your users odds are good they have a way to add a new OAuth2 client. It may be labeled as "app", "integration", or "service provider". You may need to ask their support team to enable it for your instance.
+## Step 1 - User Management
+SSO only identifies who is sitting at the computer. Creating their user record and setting permissions need to be done as a separate step.
 
-Once an OAuth2 client has been created you will enter its details at [Partner Center -> Adminstration -> Single Sign On](https://partners.vendasta.com/integrations/sso).
+Vendasta provides several options for Managing Users
 
+**1) User pre-syncing using REST APIs**
+- [Create, Update, and Delete](https://developers.vendasta.com/platform/9ae19216ae422-create-user) 
+- [User Account Association](https://developers.vendasta.com/platform/7b68c25e54818-associate-user-with-location)
 
-If you built your own site you will need to implement the Authorization, Token and UserInfo endpoints from the OAuth2 specification. Only the authorization code flow with [PKCE](https://www.oauth.com/oauth2-servers/pkce/) is required.
+**2) User pre-syncing using SCIM APIs**
+- [Create, Update, and Delete ](https://developers.vendasta.com/platform/8a121c92d9d8b-create-user)
+- [Group Management (User Account Association)](https://developers.vendasta.com/platform/e0c9197db556a-update-group)
 
-For a great into to OAuth2 see [Okta's illustrated guide](https://developer.okta.com/blog/2019/10/21/illustrated-guide-to-oauth-and-oidc).
+**3) Just In Time(JIT) User Creation**
+- Needs to be manually turned on by Vendasta 
```

Comment:
IIRC for OAuth2 SSO it is on for everyone but we didn't want to advertise that. To use it the partner needs to add a custom field to their user info endpoint that lists the AGIDs that the user should have access to.